ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -219,6 +219,30 @@ router bgp {{ router_bgp.as }}
 {%             endif %}
    {{ maximum_routes_cli }}
 {%         endif %}
+{%         if peer_group.missing_policy is arista.avd.defined %}
+{%             for direction in ['in', 'out'] %}
+{%                 set dir = 'direction_' ~ direction %}
+{%                 set policy = peer_group.missing_policy[dir] %}
+{%                 if policy.action is arista.avd.defined and policy.address_family_all is arista.avd.defined(true) %}
+{%                     set missing_policy_cli = "neighbor " ~ peer_group.name ~ " missing-policy address-family all" %}
+{%                     set include = peer_group.missing_policy[dir] %}
```

Comment:
this is the same as you already set in `policy`. Also avoid using keywords like `include` as variable names.

Reply:
fixed